package org.pos.pos.Exceptions;

public class GenericErrorMessages {
    public static final String INTERNAL_ERROR = "Se produjo un error al procesar la solicitud.";
    public static final String VALIDATION_ERROR = "Los datos proporcionados no son válidos.";
    public static final String NOT_FOUND = "El recurso solicitado no fue encontrado.";
    public static final String DUPLICATE = "El recurso ya existe.";
    public static final String UNAUTHORIZED = "No tiene permisos para realizar esta acción.";
}
